tests/Http/Place/ImportPlaceRequestHandlerTest.php

```diff
@@ -5197,55 +5205,66 @@ public function testHandleWithDuplicatePreventionEnabled(Repository $aggregateRe
             ->withJsonBodyFromArray($this->getSimplePlace())
             ->build('POST');
 
+        $this->expectExceptionObject(ApiProblem::statusConflict(
+            MultipleDuplicatePlacesFound::ERROR_MSG,
+            ['instance' => $originalPlaceUri]
```

Comment:
Here `instance` is still used but the guard throws `query`, which makes me think there is still something else going on.

Reply:
This is not verified with `expectExceptionObject` I think it is better to use `AssertApiProblemTrait`

Reply:
done